### PR TITLE
remove unused imports from aggregate.d

### PR DESCRIPTION
**Pull request by thewilsonator**

### CODE REVIEW

File: src/dmd/aggregate.d [hunk 1]
```diff
@@ -30,10 +29,8 @@ import dmd.globals;
 import dmd.id;
 import dmd.identifier;
 import dmd.mtype;
-import dmd.semantic2;
-import dmd.semantic3;
 import dmd.tokens;
-import dmd.typesem;
+import dmd.typesem : defaultInit;
```
Comment (thewilsonator) [inline src/dmd/aggregate.d]: I know this is not the DMD style but that is the only function from there and I think it ought to be in mtype.d anyway, which I plan to follow up in another PR.

Comment (wilzbach) [reply src/dmd/aggregate.d]: Walter approved selective top-level imports in the past. I think he just doesn't see much value in them, but from what I gathered he isn't against them. So imho it's more than fine if it's just a few symbols - especially if it's one or two.

Comment (thewilsonator) [reply src/dmd/aggregate.d]: Yeah I'm trying to figure out which ones use a single import and mark them so.

With this one specifically I think it should be in `mtype`.

Comment (wilzbach) [reply src/dmd/aggregate.d]: I don't think we should invest time in doing this manually except when writing code which adds new imports. IIRC people have already written tools for this. I think @CyberShadow has one?

**Review by jacob-carlborg — APPROVED** [review]

**Review by RazvanN7 — APPROVED** [review]

**Pull request merged**